setup.cfg

```diff
@@ -43,6 +43,10 @@ data_files =
     academic_observatory_workflows/database = academic_observatory_workflows/database/*
     academic_observatory_workflows/workflows/data = academic_observatory_workflows/workflows/data/*
 
+[entry_points]
+console_scripts =
+    academic-observatory-workflows-seed = academic_observatory_workflows.seed.seed:seed
+
```

Comment:
For a workflows project, must add a console script so that the seed function can be called from within the `seed_db.sh` script.

The pattern for the command name is `<package-name>-seed`.